tests/test_octodns_provider_edgedns.py

```diff
@@ -100,6 +100,7 @@ def test_apply(self):
             "ctok",
             "cid",
             "gid",
+            "Managed by OctoDNS.",
```

Comment:
Would be good to add checks to the `mock` down below that makes sure that the two new changelist calls are made w/the expected params.